src/env.cc

```diff
@@ -1,6 +1,8 @@
 #include "env.h"
 #include "env-inl.h"
+#include "async-wrap.h"
 #include "v8.h"
+#include "v8-profiler.h"
```

Comment:
node_internals.h, which is included by node.h, which is included by env-inl.h.  It's a bit roundabout, I can include it directly.